example/form/CAP_Adrenal.xml

```diff
@@ -871,7 +870,7 @@
       </Section>
     </ChildItems>
   </Body>
-  <Footer>
+  <Footer ID="Footer_129.1000043_18b9cd14-2d84-4c8f-8bb6-7da6db5f05c6">

```

Comment:
Footer element was missing the required ID attribute; fixed by following pattern of Body element ID attribute value (prepended with "Footer_" instead of "Body_").